src/test/java/codeu/model/store/basic/MessageStoreTest.java

```diff
@@ -1,89 +1,85 @@
 package codeu.model.store.basic;
 
-import codeu.model.data.Message;
-import codeu.model.store.persistence.PersistentStorageAgent;
-import java.time.Instant;
+import static codeu.model.data.ModelDataTestHelpers.assertMessageEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.UUID;
-import org.junit.Assert;
+
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mockito;
 
+import codeu.model.data.Message;
+import codeu.model.data.ModelDataTestHelpers.TestMessageBuilder;
+import codeu.model.store.persistence.PersistentStorageAgent;
 
 public class MessageStoreTest {
 
   private MessageStore messageStore;
   private PersistentStorageAgent mockPersistentStorageAgent;
 
   private final UUID CONVERSATION_ID_ONE = UUID.randomUUID();
-  private final Message MESSAGE_ONE =
-      new Message(
-          UUID.randomUUID(),
-          CONVERSATION_ID_ONE,
-          UUID.randomUUID(),
-          "message one",
-          Instant.ofEpochMilli(1000));
-  private final Message MESSAGE_TWO =
-      new Message(
-          UUID.randomUUID(),
-          CONVERSATION_ID_ONE,
-          UUID.randomUUID(),
-          "message two",
-          Instant.ofEpochMilli(2000));
-  private final Message MESSAGE_THREE =
-      new Message(
-          UUID.randomUUID(),
-          UUID.randomUUID(),
-          UUID.randomUUID(),
-          "message three",
-          Instant.ofEpochMilli(3000));
 
   @Before
   public void setup() {
     mockPersistentStorageAgent = Mockito.mock(PersistentStorageAgent.class);
     messageStore = MessageStore.getTestInstance(mockPersistentStorageAgent);
-
-    final List<Message> messageList = new ArrayList<>();
-    messageList.add(MESSAGE_ONE);
-    messageList.add(MESSAGE_TWO);
-    messageList.add(MESSAGE_THREE);
-    messageStore.setMessages(messageList);
   }
 
   @Test
   public void testGetMessagesInConversation() {
+    final Message message1 = new TestMessageBuilder().conversation(CONVERSATION_ID_ONE).build();
```

Comment:
While we're here, maybe add comments to these tests? Probably not something we'd do in Real Google Code ™, but if the goal is to make these as easy to understand as possible, I think a few comments would go a long way. This is probably something I should have added from the start, so if you want to keep this PR focused on your structural changes, that's cool too.

Reply:
Great point.